bin/disturb.php

```diff
@@ -1,74 +1,82 @@
+
+try {
+    /**
+     * Register the autoloader and tell it to register the tasks directory
+     */
+    $loader = new \Phalcon\Loader();
+    $loader->registerNamespaces(
+        [
+            'Vpg\Disturb' => realpath(__DIR__ . '/../Library/')
+        ],
+        true
+    );
 
-/**
- * Register the autoloader and tell it to register the tasks directory
- */
-$loader = new \Phalcon\Loader();
-$loader->registerNamespaces(
-    [
-        'Vpg\Disturb' => realpath(__DIR__ . '/../Library/')
-    ],
-    true
-);
+    $loader->registerFiles([
+        __DIR__ . '/../../vendor/autoload.php',
+        __DIR__ . '/../../../autoload.php'
+    ]);
+    $loader->register();
 
-$loader->registerFiles([
-    __DIR__ . '/../../vendor/autoload.php',
-    __DIR__ . '/../../../autoload.php'
-]);
-$loader->register();
+    require_once(__DIR__ . '/../Library/Core/DI.php');
 
-require_once(__DIR__ . '/../Library/Core/DI.php');
+    $di->setShared('loader', $loader);
 
-$di->setShared('loader', $loader);
+    // Create a console application
+    $console = new ConsoleApp();
+    $console->setDI($di);
 
+    /**
+     * Process the console arguments
+     */
+    $arguments = [];
 
-// Create a console application
-$console = new ConsoleApp();
-$console->setDI($di);
+    foreach ($argv as $k => $arg) {
+        if ($k === 1) {
+            $arguments['task'] = $arg;
+        } elseif ($k === 2) {
+            $arguments['action'] = $arg;
+        } elseif ($k >= 3) {
+            $arguments['params'][] = $arg;
+        }
+    }
 
-/**
- * Process the console arguments
- */
-$arguments = [];
+    // Load client boostrap file
+    $paramHash = ConsoleApp::parseLongOpt(join($arguments['params'], ' '));
 
-foreach ($argv as $k => $arg) {
-    if ($k === 1) {
-        $arguments['task'] = $arg;
-    } elseif ($k === 2) {
-        $arguments['action'] = $arg;
-    } elseif ($k >= 3) {
-        $arguments['params'][] = $arg;
+    $workflowConfigDto = WorkflowConfigDtoFactory::get($paramHash['workflow']);
+    $projectBootstrapFilePath = $workflowConfigDto->getProjectBoostrapFilepath();
+    if (is_readable($projectBootstrapFilePath)) {
+        $di->get('logr')->info('Loading Bootstrap : ' . $projectBootstrapFilePath);
+        require_once($projectBootstrapFilePath);
     }
-}
 
-// Load client boostrap file
-$paramHash = ConsoleApp::parseLongOpt(join($arguments['params'], ' '));
-$workflowConfig = new Json($paramHash['workflow']);
-$projectBootstrapFilePath = $workflowConfig['projectBootstrap'] ?? '';
-if (is_readable($projectBootstrapFilePath)) {
-    $di->get('logr')->info('Loading Bootstrap : ' . $projectBootstrapFilePath);
-    require_once($projectBootstrapFilePath);
-}
-$di->get('disturb-config')->workflowConfigFilePath = $paramHash['workflow'];
+    $di->get('disturb-config')->workflowConfigFilePath = $paramHash['workflow'];
 
-try {
     // Handle incoming arguments
     $console->handle($arguments);
```

Comment:
sound this var has been removed ...
did you run a test workflow successfully w/ this code ?